core/kazoo_data/src/kzs_db.erl

```diff
@@ -222,9 +222,17 @@ add_update_remove_views(Server, Db, CurrentViews, NewViews, ShouldRemoveDangling
 add_views(Server, Db, Add, NewViews) ->
     Views = [props:get_value(Id, NewViews) || Id <- Add],
     {'ok', JObjs} = kzs_doc:save_docs(Server, Db, Views, []),
-    [kz_doc:id(JObj)
+    [Id
```

Comment:
Can this list comprehension be put in a function; looks identical to the LC below this one.